src/ProjectSystemTools/BuildLogging/Model/Frontend/IFrontendBuildTableDataSource.cs

```diff
@@ -17,6 +16,6 @@ public interface IBuildTableDataSource : ITableDataSource
 
         void Clear();
 
-        ILogger CreateLogger(bool isDesignTime);
+        string RetrieveLogForBuild(int buildID);
```

Comment:
This method returns a path. Two points:

1. Consider using "Path" in the name, as it's not clear whether it returns the log contents for example.
2. The path won't be useful on the front end. Instead you'll likely need a method on the RPC service interface to stream the contents of the log from server to client, so the client can save the file to disk somewhere.

Reply:
For this API, I'll definitely need to change it once I move to working on async server client connection (where the code is not running on the same machine)
I decided to leave this as a workaround that will allow project-system-tools to have all features still working,
but will change this in a later PR with the async changes.
Do you think this is an OK approach for now?